apps/directory/src/property-config/initialLandingpage.js

```diff
@@ -63,7 +63,7 @@ const initialLandingpage = {
   css: {
     pageHeader: {
       backgroundStyle:
-        'background: url("https://github.com/molgenis/molgenis-app-biobank-explorer/blob/master/public/img/bacteria.jpg?raw=true");background-size: cover;height: 30rem;width: 75%;border-radius: 1rem; color: #fff;text-align: center;',
+        'background: url("https://github.com/molgenis/molgenis-emx2/blob/master/apps/directory/public/img/bacteria.jpg?raw=true");background-size: cover;height: 30rem;width: 75%;border-radius: 1rem; color: #fff;text-align: center;',
```

Comment:
I would prefer to use a internally stored file, i.e., one that is served via the 'apps' folder i.e. using address `/directory/public/img/bacteria.jpg`. Otherwise this breaks if we would change github or if it is run behind a firewall or offline.

Reply:
I think we should do it like Morris indicated, that is the more future proof way and if I look at the ticket is already indicates that the files are in the public/img/ directory, which is already being served.
e.g. https://directory.bbmri-eric.eu/public_html/apps/directory/img/bacteria.jpg works